lib/Dialect/FIRRTL/Transforms/CMakeLists.txt

```diff
@@ -3,6 +3,7 @@ add_circt_dialect_library(CIRCTFIRRTLTransforms
   BlackBoxReader.cpp
   CheckCombCycles.cpp
   CheckCombLoops.cpp
+  DropConst.cpp
```

Comment:
alphabetical please

Reply:
Oops, this was due to a rename. Updated.